app/src/main/java/dev/arkbuilders/arkmemo/ui/views/NotesCanvas.kt

```diff
@@ -0,0 +1,70 @@
+package dev.arkbuilders.arkmemo.ui.views
+
+import android.content.Context
+import android.graphics.Canvas
+import android.graphics.Path
+import android.util.AttributeSet
+import android.view.MotionEvent
+import android.view.View
+import dev.arkbuilders.arkmemo.graphics.SVGCommand
+import dev.arkbuilders.arkmemo.ui.viewmodels.DrawPath
+import dev.arkbuilders.arkmemo.ui.viewmodels.GraphicNotesViewModel
+
+class NotesCanvas(context: Context, attrs: AttributeSet): View(context, attrs) {
+
+    private var currentX = 0f
+    private var currentY = 0f
+    private lateinit var viewModel: GraphicNotesViewModel
+    private var path = Path()
+
+    override fun onSizeChanged(w: Int, h: Int, oldw: Int, oldh: Int) {
+        super.onSizeChanged(w, h, oldw, oldh)
+        viewModel.svg().setViewBox(w.toFloat(), h.toFloat())
+    }
```

Comment:
Just curious, can the size really change? Does viewbox of the resulted SVG depend on screen dimension or layout?

Reply:
It depends on layout